ocm/__main__.py

```diff
@@ -255,20 +288,64 @@ def main():
     create_parser.set_defaults(callable=create)
 
     add_parser = maincmd_parsers.add_parser(
+        'add',
+    )
+    add_subparsers = add_parser.add_subparsers(
+        title='add ocm references',
+        required=True,
+    )
+    add_cref_parser = add_subparsers.add_parser(
+        'component-ref',
+        help='add component-reference',
+    )
+
+    add_cref_parser.add_argument('--file', '-f', required=True)
+    add_cref_parser.add_argument(
+        '--name', '-n',
+        help='name of the component-reference, defaults to last part of component-name',
+    )
+    add_cref_parser.add_argument(
+        '--component-name', '-c',
+        dest='cname',
```

Comment:
preferably do not abbreviate. we commonly use argparse's default-naming-scheme (with some exceptions, e.g. pluralising if action is set to `append`);